elixir/apps/domain/lib/domain/auth/identity/sync.ex

```diff
@@ -151,4 +158,9 @@ defmodule Domain.Auth.Identity.Sync do
       end
     end)
   end
+
+  defp add_email_attr(attrs) do
+    email = attrs["provider_state"]["userinfo"]["email"]
```

Comment:
Probably don't need to worry about this, but is there an edge case where the email provided by the IdP isn't valid? I.e. do we need to apply valid_email? to it?

Reply:
Like you said, it probably isn't necessary, but I did put a check around this code for a valid email.